lib/rspec/support/ruby_features.rb

```diff
@@ -45,7 +45,7 @@ def mri?
     # Provides query methods for ruby features that differ among
     # implementations.
     module RubyFeatures
-      module_function
+    module_function
```

Comment:
They don't have the same meaning. `module_function` makes the methods available both as a method on the module and adds the methods as `private` when `include` in another class or module.